confgenerator/testdata/valid/windows_logging-no_conf/input.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC
```

Comment:
Why are the golden files for this test empty?

Reply:
'cause the logging is not defined thus it's expected being empty. See also here:  https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/valid/logging-no_conf/golden_fluent_bit_main.conf